### PR TITLE
[bitnami/discourse] Add VIB tests

### PR DESCRIPTION
**Pull request by migruiz4**
### Description of the change

The main objective of this PR is to publish our Bitnami Discourse container using VMware Image Builder. In order to do that, several changes are included:

- Increasing the existing test coverage of the asset by adding Goss tests.
- Update verify and publish VIB pipeline's definitions.

### Benefits

- Ensuring higher quality of the container catalog.
- Increased pool of assets completely handled by VMware Image Builder.

### Possible drawbacks

Automated tests could introduce additional flakiness to the CI/CD.

### Applicable issues

NA

### CODE REVIEW

File: .vib/discourse/goss/discourse.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+command:
```
Comment (FraPazGal) [inline .vib/discourse/goss/discourse.yaml:1]: I would add several checks related to the logic executed at compilation time:

- Check the global npm package `terser` installed
- Check Passenger native support works is properly installed

File: .vib/discourse/goss/vars.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+binaries:
+  - wait-for-port
+  - python
+  - ruby
+  - node
+  - brotli
+  - psql
```
Comment (FraPazGal) [inline .vib/discourse/goss/vars.yaml:7]: I would add bundle, which is used as a discourse cli.

**Review by FraPazGal — COMMENTED** [review]
Thanks PR @migruiz4! I added a few comments, please review them whenever you can.

File: .vib/discourse/goss/discourse.yaml (hunk 1 above)
Comment (migruiz4) [reply .vib/discourse/goss/discourse.yaml:1]: I added the `terser` node module test:
```
  check-terser-modules:
    exec: npm list -g
    exit-status: 0
    stdout:
      - terser
```

Is it okay or would it be enough if we simply add it to the list of checked binaries (as it should be in the PATH)? Maybe both?

Comment (FraPazGal) [reply .vib/discourse/goss/discourse.yaml:1]: I think the node module test is great 👍

**Review by FraPazGal — APPROVED** [review]

**Pull request merged**